### DISCUSSION TITLE
Newbie question.

### DISCUSSION THREAD

**Discussion by Poptoogi** [keiwando/toonsquid#587] [General]
I just started using ToonSquid and I love it but I'm having a problem with something. I can draw a vector line and can edit vertices if i make it a straight line or a curve but I can't edit points if it's just a drawn line. Is this a limitation to the software or am I just not finding the solution within the app? If anyone knows, let me know! Thanks! :)

**Comment by Cat-Butler** [comment]
Currently a limitation, if I'm understanding you correctly.
"Path" objects are standard vector shapes/curves as you commented.

However, "vector brushes" are more like a pixel brush that is automatically auto-traced. You can see this more clearly with complex vector brushes - they draw in pixel mode, then convert to vector when you finish the stroke (see below).
It is a brilliant feature (great for zoom effects/keyframes), but is not what you might think of as a standard vector line/curve.

https://github.com/keiwando/toonsquid/assets/25494772/a1e8f421-776b-4c27-89da-b6a1aa4b6ae8

Note that you do have the Procreate-style [Shape Assist](https://toonsquid.com/handbook/brushes/shape_assist/) functionality for all brushes, pixel and vector (while still in pixel mode) - draw your line and hold (not releasing your Pencil/finger) at the end of the line. You will then have the option (icon at top of the screen) to edit the vertices of the line - currently works for partial ellipses, simple arcs, straight lines and straight-segmented polylines, and a few closed shapes (ellipses, polygons) but not arbitrary drawn curves, unfortunately.
